packages/minifier-js/minifier.js

```diff
@@ -1,14 +1,14 @@
 let terser;
 
-const terserMinify = async (source, options, callback) => {
+const terserMinify = async (source, options, callback = () => {}) => {
   terser = terser || Npm.require("terser");
   try {
     const result = await terser.minify(source, options);
     callback(null, result);
     return result;
   } catch (e) {
     callback(e);
-    return e;
+    throw e;
```

Comment:
That's a rather big change - where is this coming from? If it's only because `meteorJsMinifyAsync` needs that, maybe it'd be easier to inline it instead? Removing the synchronous version will be easier then.

Reply:
You're right, actually it was a mistake. The behavior should be similar to the synchronous method, so it should just return the error. But I think the inline approach is better since we are planning to remove it later.